front-end/src/ui-components/CouncilSignalBar.tsx

```diff
@@ -0,0 +1,74 @@
+// Copyright 2019-2020 @paritytech/polkassembly authors & contributors
+// This software may be modified and distributed under the terms
+// of the Apache-2.0 license. See the LICENSE file for details.
+
+import styled from '@xstyled/styled-components';
+import React from 'react';
+
+interface Props {
+	className?: string
+	ayeSignals: number
+	naySignals: number
+	councilSize: number
+}
+
+const CouncilSignalBar = ({ ayeSignals, className, councilSize, naySignals }: Props) => {
+	const notSignaled = councilSize - ayeSignals - naySignals;
+
+	return (
+		<div className={className}>
+			<div className='ayes'>{ayeSignals === 1 ? '1 Aye' : `${ayeSignals} Ayes`}</div>
```

Comment:
This should be `< 2`, because 0 Aye also shouldn't have any "s" (here and in any other place).
```suggestion
			<div className='ayes'>{ayeSignals} {ayeSignals < 2 ? 'Aye' : `Ayes`}</div>
```